### ISSUE TITLE
Mind Vision Bugs. Resisting on same-faction and not allowing "hops"

### ISSUE THREAD

**Issue by Tabletopcheese** [LightsHope/issues#654]
You should be able to hop from target to target while channeling mind vision. It also should not resist on same faction. (Honestly, it should not even have a high chance to resist at all, you can MV bosses at 60 and only 1/100 would resist)

**Comment by Cabro** [comment]
Can confirm on the hop thing. Also, I was not aware you can resist the mind vision here? Why is this?
Question @Tabletopcheese : can you mind vision people away from normal vision range? Can you mind vision someone from Winterspring while you are in... Tanaris let's say? You could totally do that in vanilla on retail ;)

**Comment by Tabletopcheese** [comment]
@Cabro - The distance works fine. There is just no way to refresh it. Followed someone from a flight path for the duration.

It could be a problem with channeling spells. Such as MC.

My other ticket foor reference https://github.com/LightsHope/issues/issues/624

**Comment by Cabro** [comment]
No, i mean: can you start channeling Mind Vision on a party/raid member that is out of your vision range? Not if you start on someone close and he goes out of it.

**Comment by Tabletopcheese** [comment]
@Cabro Yes. It is not allowing you to cast on party while *in range* (technically they arent out of range, it being 50000) 

But when targeting a party member it does not allow you as well. Which is another bug
